app/models/miq_server/worker_management/monitor/stop.rb

```diff
@@ -10,19 +10,19 @@ def clean_stop_worker_queue_items
     ).destroy_all
   end
 
-  def stop_worker_queue(worker, monitor_status = :waiting_for_stop, monitor_reason = nil)
+  def stop_worker_queue(worker, monitor_reason = nil)
     MiqQueue.put_deprecated(
       :class_name  => self.class.name,
       :instance_id => id,
       :method_name => 'stop_worker',
-      :args        => [worker.id, monitor_status, monitor_reason],
+      :args        => [worker.id, monitor_reason],
       :queue_name  => 'miq_server',
       :zone        => zone.name,
       :server_guid => guid
     )
   end
 
-  def stop_worker(worker, monitor_status = :waiting_for_stop, monitor_reason = nil)
+  def stop_worker(worker, monitor_reason = nil)
```

Comment:
Do we need to worry about existing queued items using the old interface?  I don't remember who calls `stop_worker_queue`

Reply:
For this, no. On server start there is [a call to `clean_stop_worker_queue_items`](https://github.com/ManageIQ/manageiq/blob/6e473debac2fe9e03e8649f37cac5569a3bfc7a1/app/models/miq_server.rb#L137).

As a side note. Is this generally something we worry about? Couldn't there theoretically be a queue message for any public method? Do we really try to deal with possible existing queue items every time we change a method interface?